scripts/check-config-examples.sh

```
@@ -176,13 +233,55 @@ probe() {
+        if [ "$intent" = "fragment" ]; then
+            local expected_fragment_error
+            expected_fragment_error="$(fragment_expected_error "$label")"
+            if [ -z "$expected_fragment_error" ]; then
+                failed=$((failed+1))
+                echo "  ✗ $label" >>"$FAILURES"
+                echo "      undeclared fragment exemption; add its exact expected refusal to fragment_expected_error" >>"$FAILURES"
+                return 0
+            fi
+            if ! grep -Fqi "$expected_fragment_error" "$check_out"; then
+                failed=$((failed+1))
+                echo "  ✗ $label" >>"$FAILURES"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)